src/utils/word-api.ts

```diff
@@ -151,7 +151,15 @@ class WordApi {
           const tag = JSON.parse(
             citation.tag.substring(16)
           ) as StatefulCitation;
-          return tag.citationItems.map((item) => item.id);
+          return tag.citationItems.map((item) => ({
+            id: item.id,
```

Comment:
Again the question if it's really necessary to map items here since you don't change anything and is the same property names.

Reply:
This map is just filtering out some extra stuff.